src/commands/e2e/WdioConfigGenerator.ts

```diff
@@ -1,25 +1,37 @@
 import {ConfigTemplate} from './ConfigTemplate';
 import * as path from 'path';
 import * as fs from 'fs';
+import {E2EEnvTemplate} from './E2EEnvTemplate';
 
 export class WdioConfigGenerator {
     public static readonly WDIO_CONF_NAME: string = 'wdio.conf.js';
+    public static readonly E2E_ENV_NAME: string = 'e2e.ts';
     private readonly baseUrl: string;
+    private readonly context: string;
+    private readonly tenantId: string = '';
     private readonly plugins: string[];
     private readonly browser: string;
     private readonly timeout: number;
     private readonly workingDir: string;
     private readonly headless: boolean;
 
-    constructor(plugins: string[], baseUrl: string, browser: string, timeout: number, workingDir: string, headless: boolean) {
+    constructor(plugins: string[], baseUrl: string, browser: string, context: string, tenantId: string, timeout: number, workingDir: string, headless: boolean) {
         this.browser = browser;
         this.plugins = plugins;
         this.baseUrl = baseUrl;
+        this.context = context;
+        this.tenantId = tenantId;
         this.timeout = timeout;
         this.workingDir = workingDir;
         this.headless = headless;
     }
 
+    public generateE2EEnv(): void {
+        const e2eFolder = this.pathToE2EFolder('cf.cplace.platform');
```

Comment:
This would not work in a repo other than the main repository. The platform plugin does not exist there. Here we have to explicitly construct the path to the platform plugin with the relative path to the main repository.